### DISCUSSION TITLE
GitHub Education access for distance learning students without university email

### DISCUSSION THREAD

**Discussion by RyuintheMoon** [community/community#130776] [GitHub Education]
### Hi everyone,

I'm a university student in Japan. My university has both on-campus and distance learning departments, but only on-campus students are provided with university email addresses. However, to register for GitHub Education, an email address is required, which makes it difficult for me to get verified.

Is there any way for distance learning students to use Copilot for free? 

Has anyone else encountered a similar situation? How did you resolve it?

**Comment by the-deadrose** [comment]
GitHub Education offers alternative verification methods, like uploading a student ID or other official paperwork. Be sure to review all the verification choices on the GitHub Education site.
